sdks/python/apache_beam/examples/complete/tfidf.py

```diff
@@ -35,6 +34,12 @@
 from apache_beam.options.pipeline_options import SetupOptions
 from apache_beam.pvalue import AsSingleton
 
+# Protect against environments where gcsio library is not available.
+try:
+  from apache_beam.io.gcp import gcsio
+except ImportError:
+  gcsio = None
```

Comment:
Does this actually protect us? It looks like this would just change the error to a more confusing one: `None has no attribure GcsIO`, when gcsio is used. I think just letting the ImportError raise would be preferable.

Alternatively we could add a `skipIf(gcsio is None)`, but that might lead to us unintentionally skipping it indefinitely.